pos-print/src/main/java/com/clouway/pos/print/PosPrintService.java

```diff
@@ -39,9 +40,13 @@ public static void main(String[] args) {
       new PersistentModule(client, commandCLI.dbName())
     );
 
+
     HttpBackend backend = new HttpBackend(commandCLI.httpPort(), injector);
     backend.start();
 
+    ReceiptPrintWorker backgroundPrinter = injector.getInstance(ReceiptPrintWorker.class);
```

Comment:
ReceiptPrintingService sounds better then worker.

Reply:
Renamed to ReceiptPrintingService.